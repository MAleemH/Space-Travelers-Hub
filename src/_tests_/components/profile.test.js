import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Profile from '../../components/profile/profile';
import store from '../../redux/store';

describe('Profile', () => {
  it('Profile Page Rendered Correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Profile />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
