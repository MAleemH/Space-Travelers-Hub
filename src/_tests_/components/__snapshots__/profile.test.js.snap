// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Profile Page Rendered Correctly 1`] = `
<div
  className="reserved-lists"
>
  <div
    className="missions-section"
  >
    <p
      className="profile-titles"
    >
      My Missions
    </p>
    <table
      className="missionList"
    >
      <tr
        className="reservedM"
      >
        <p
          className="no-items"
        >
          No Missions Joined
        </p>
      </tr>
    </table>
  </div>
  <div
    className="rockets-section"
  >
    <p
      className="profile-titles"
    >
      My Rockets
    </p>
    <table
      className="rockets-list"
    >
      <tr
        className="rockets-list-items"
      >
        <p
          className="no-items"
        >
          No Rockets Reserved
        </p>
      </tr>
    </table>
  </div>
</div>
`;
